src/app/examples/examples.common.ts

```diff
@@ -7,11 +7,11 @@ import { ChartExampleComponent } from './chart-example/chart-example.component';
 import { FontsExampleComponent } from './fonts-example/fonts-example.component';
 
 export const COMPONENT_DECLARATIONS: any[] = [
-  ButtonExampleComponent,
-  CardExampleComponent,
-  ListExampleComponent,
-  GridExampleComponent,
-  AvatarExampleComponent,
-  ChartExampleComponent,
-  FontsExampleComponent,
+    ButtonExampleComponent,
```

Comment:
Looks like you have you tab space set to 4 and not 2. 

I am unsure what is default for the Kirby project?

Reply:
I think it is 2. Now fixed. I will notify @askarby